src/View/Sidebar/NetworkListBox.vala

```diff
@@ -41,21 +41,49 @@ public class Sidebar.NetworkListBox : Gtk.ListBox, Sidebar.SidebarListInterface
                 select_item ((SidebarItemInterface) row);
             }
         });
+
+        set_sort_func (network_sort_func);
+    }
+
+    private int network_sort_func (Gtk.ListBoxRow? row1, Gtk.ListBoxRow? row2) {
+        string key1 = row1 != null ? ((AbstractMountableRow)row1).sort_key : "";
+        string key2 = row2 != null ? ((AbstractMountableRow)row2).sort_key : "";
```

Comment:
Shouldn't we also check for the type here? Also, we can simply use `var`:

```
var key 1 = row1 != null && (row1 is AbstractMountableRow) ? ...
...
```

Reply:
As we are in full control of what rows are added to this listbox I am not sure a check is necessary but I will put one in just to be on the safe side.